src/background/sidebar-injector.js

```diff
@@ -189,6 +187,7 @@ export function SidebarInjector() {
 
     const canInject = await canInjectScript(tab.url);
     if (canInject) {
+      /** @type {{type: string}[]}} */
       const frameResults = await chromeAPI.tabs.executeScript(tab.id, {
         code: toIIFEString(detectContentType),
       });
```

Comment:
The type of `result` variable can be correctly inferred.
 
![Screenshot 2021-09-27 at 15 22 56](https://user-images.githubusercontent.com/8555781/134917051-533d8dc9-afd6-4a17-9e48-a7a6c9a98c40.png)